packages/react-components/react-virtualizer/src/components/Virtualizer/Virtualizer.types.ts

```diff
@@ -0,0 +1,128 @@
+import type { ReactNode, MutableRefObject } from 'react';
```

Comment:
```suggestion
import * as React from 'react';
```

We generally do the start import in the repo since it's friendlier with typescript